library/src/kinectWrapper_server.cpp

```diff
@@ -125,11 +127,13 @@ bool KinectWrapperServer::open(const Property &options)
     seatedMode=opt.check("seatedMode");
     img_width=opt.check("img_width",Value(320)).asInt();
     img_height=opt.check("img_height",Value(240)).asInt();
+    depth_width=opt.check("depth_width",Value(320)).asInt();
+    depth_height=opt.check("depth_height",Value(240)).asInt();
 
-    buf=new unsigned short[KINECT_TAGS_DEPTH_WIDTH*KINECT_TAGS_DEPTH_HEIGHT];
-    bufPl=new unsigned short[KINECT_TAGS_DEPTH_WIDTH*KINECT_TAGS_DEPTH_HEIGHT];
-    bufF=new float[KINECT_TAGS_DEPTH_WIDTH*KINECT_TAGS_DEPTH_HEIGHT];
-    bufFPl=new float[KINECT_TAGS_DEPTH_WIDTH*KINECT_TAGS_DEPTH_HEIGHT];
+    buf=new unsigned short[depth_width*depth_height];
```

Comment:
Hi @pattacini,
in the server, the `delete`'s are found in `threadRelease`. However, I forgot to free the memory in the client, which is addressed in https://github.com/Tobias-Fischer/kinect-wrapper/commit/eb791a30d8d73e5b20b33c1e1fae1fba53300b4a

Thanks for pointing that out!